### PR TITLE
feat: Website headers renamed to align with the code

### LINKED ISSUE #48: Move infrastructure page to Blueprints tab
- Move https://awslabs.github.io/ai-on-eks/docs/ai/infrastructure page under https://awslabs.github.io/ai-on-eks/docs/blueprints/ai-ml

- Rename the `Blueprints` section in the webpage header to `Infra` to align with the naming in the code repository.
- Rename the `AI` section in the webpage header to `Blueprints` to match the structure and naming in the code repository.
- The new `Infra` section on the webpage will host Terraform based infrastructure templates, including base setups and components like NVIDIA NIM, Dynamo, RAPIDS etc.
- The `Blueprints` section will now showcase end-to-end deployment patterns for training, fine-tuning, inference, agentic workflows, and more.

#### What is the outcome that you are trying to reach?

- Idea is to align the naming for both Website and the code repo
- Will use Blueprints and Infra a new terms

### PR DESCRIPTION
**Pull request by vara-bonthu**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction. When we triage the issues, we will add labels to the issue like "Enhancement", "Bug" which should indicate to you that this issue can be worked on and we are looking forward to your PR. We would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/ai-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

Closes #48 

**Change1:**
image

image

**Change2:**
image

**Change3:**

image

### More

- [x] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ ] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by omrishiv** [comment]
What do you think of swapping infrastructure and blueprints? We'd need to deploy before jumping in to the blueprints

**Review by omrishiv — CHANGES_REQUESTED** [review]
Tests are reporting broken links, can you fix please?

**Comment by omrishiv** [comment]
Can you check what's going on with nvidia nim? I think you may have accidentally added the base infrastructure into there

File: website/docs/infra/ai-ml/emr-spark-rapids.md [hunk 1]
```diff
@@ -199,7 +200,7 @@ Here, we will utilize a helper shell script to execute the job. This script requ
 This script will ask for certain inputs that you can obtain from Terraform outputs. See the example below.
 
 ```bash
-cd ai-ml/emr-spark-rapids/examples/xgboost/ && chmod +x execute_spark_rapids_xgboost.sh
+cd ai-on-eks/infra/emr-spark-rapids/examples/xgboost/ && chmod +x execute_spark_rapids_xgboost.sh
```
Comment (omrishiv) [inline website/docs/infra/ai-ml/emr-spark-rapids.md:203]: Is this the right path?

**Comment by vara-bonthu** [comment]
> Can you check what's going on with nvidia nim? I think you may have accidentally added the base infrastructure into there

Updated the existing commit to remove the files

File: website/docs/infra/ai-ml/emr-spark-rapids.md (hunk 1 above)
Comment (vara-bonthu) [reply website/docs/infra/ai-ml/emr-spark-rapids.md:203]: Yes, that's the correct path

Comment (omrishiv) [reply website/docs/infra/ai-ml/emr-spark-rapids.md:203]: should it just be `cd infra/emr-spark-rapids...` without the `cd ai-on-eks`? We assume we're already in the root at this point

File: website/docs/infra/ai-ml/jupyterhub.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 sidebar_label: JupyterHub on EKS
+sidebar_position: 2
```
Comment (omrishiv) [inline website/docs/infra/ai-ml/jupyterhub.md:3]: Do we need all the hardcoded `sidebar_positions`? This will mean moving things around if we need to reorder. I think it might make sense to highlight the first, then let everything else be alphabetical?

Comment (vara-bonthu) [reply website/docs/infra/ai-ml/jupyterhub.md:3]: I had to do this to make sure we can fix the order as I want `JARK` to be on the top.

Comment (omrishiv) [reply website/docs/infra/ai-ml/jupyterhub.md:3]: Jark should be 1, that's fine, but if we omit the `sidebar_position` from the rest, they should alphabetize, no?

File: website/docs/infra/ai-ml/emr-spark-rapids.md (hunk 1 above)
Comment (vara-bonthu) [reply website/docs/infra/ai-ml/emr-spark-rapids.md:203]: I noticed that, but for consistency with the rest of the document which references paths from the home folder(`ai-on-eks`) hence I updated the path to start from `ai-on-eks` for this step as well.

File: website/docs/infra/ai-ml/jupyterhub.md (hunk 2 above)
Comment (vara-bonthu) [reply website/docs/infra/ai-ml/jupyterhub.md:3]: Not sure about that. Let me try that now

Comment (vara-bonthu) [reply website/docs/infra/ai-ml/jupyterhub.md]: I think that worked. Only added index intro and jark

File: website/docs/infra/ai-ml/emr-spark-rapids.md (hunk 1 above)
Comment (omrishiv) [reply website/docs/infra/ai-ml/emr-spark-rapids.md:202]: We assume we're already at `ai-on-eks` for the purpose of all documentation. Shouldn't this be: 
```
cd infra/emr-spark-rapids/examples/xgboost/ && chmod +x execute_spark_rapids_xgboost.sh
```

?

Comment (vara-bonthu) [reply website/docs/infra/ai-ml/emr-spark-rapids.md:202]: Not in this doc. This is written differently. Checkout the entire doc on how the flow is going. There are no dependencies on previous steps.

Comment (vara-bonthu) [reply website/docs/infra/ai-ml/emr-spark-rapids.md:202]: Let me modify all the steps

File: website/sidebars.js [hunk 3]
```diff
@@ -16,10 +16,9 @@ const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
   // docs: [{type: 'autogenerated', dirName: '.'}],
   // But you can create a sidebar manually
-    ai: [{type: 'autogenerated', dirName: 'ai'}],
-    blueprints: [{type: 'autogenerated', dirName: 'blueprints'}],
+    blueprints: [{type: 'autogenerated', dirName: 'blueprints'}],
```
Comment (omrishiv) [inline website/sidebars.js:19]: can we remove the whitespace after the `,`

**Review by omrishiv — APPROVED** [review]
Approved, thanks for spending the time updating. Can you please remove the whitespace in the comment before merge. Thanks!

File: website/sidebars.js (hunk 3 above)
Comment (vara-bonthu) [reply website/sidebars.js]: Let me run the npx command here for now. I will raise another PR for adding this as Github workflow for linting with npx
